extensions/wikia/CommunityPage/styles/components/_moderators-module.scss

```diff
@@ -0,0 +1,60 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+@import 'variables';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
+
+// Styles for the moderators module in-page
+$moderator-module-space-small: 6px;
+$moderator-module-space-medium: 2 * $moderator-module-space-small;
+
+.moderators-module {
+	.avatar-container {
+		display: inline-block;
+		margin-right: $moderator-module-space-small;
+		vertical-align: middle;
+	}
+
+	.community-page-moderators-module-section {
+		margin-top: $moderator-module-space-medium;
+
+		&:first-of-type {
+			margin-top: 0;
+		}
+
+		.top-moderators {
+			margin-left: 0;
+		}
+	}
+
+	.community-page-moderator-details {
+		vertical-align: middle;
+	}
+
+	.community-page-moderators-list-item {
+		@include flexbox();
+		@include align-items(flex-start);
+
+		padding: $moderator-module-space-medium 0 0;
```

Comment:
Haven't checked that, may be equivalent - will check and fix if the result is the same